python/snewpy/snowglobes_interface.py

```diff
@@ -284,5 +282,109 @@ def run(self):
             return tables
         else:
             raise RuntimeError('SNOwGLoBES run failed:\n'+stderr)
+
+class SimpleRate(SNOwGLoBES):
```

Comment:
Oh, if you don't reuse the `__init__` method from `SNOwGLoBES`, there is no reason to subclass SNOwGLoBES Sorry, I didn't realize in the first place, when I suggested you to use subclass. In the current implementation, these two classes don't have any shared methods, so subclassing has no effect.

Reply:
Actually I am using the _load_detectors and _load_channels methods of the SNOwGLoBES  class. Do you think I should just add them to the SimpleRate class and drop the inheritance?

Reply:
You're right, sorry, I missed it. Then it's good and completely adequate as it is